src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -310,6 +310,22 @@ function stanley_reisner_ideal(R::MPolyRing, K::SimplicialComplex)
     return ideal([ R([1], [_characteristic_vector(f,n)]) for f in minimal_nonfaces(K) ])
 end
 
+
+@doc Markdown.doc"""
+    stanley_reisner_ideal(R::MPolyRing{fmpq}, K::SimplicialComplex)
+
+Return the Stanley-Reisner ideal of the abstract simplicial complex `K` in the given ring `R`.
+This method is tailor-made to accept the graded Cox ring of a toric variety.
+
```

Comment:
No new line or an example :)

Reply:
Ok. I will adjust this upon resolving the points mentioned below.